src/index.js

```diff
@@ -0,0 +1,36 @@
+import TaskList from './taskList.js';
+import './style.css';
+import Task from './task.js';
+import './interaction.js';
+
+const tasks = (localStorage.tasks ? JSON.parse(localStorage.tasks) : []);
+
+const list = new TaskList(tasks);
+window.addEventListener('load', () => {
+  list.show();
+});
+
+const addTask = () => {
+  const txt = document.querySelector('.txtNew');
+  const description = txt.value;
+  if (description === '') {
+    alert('Please type your task description');

```

Comment:
Try to avoid using the alert method. It could stop the normal execution of your code. Instead, try to print the message below or above the input.

Reply:
An alternative approach implemented.